package.json

```diff
@@ -19,13 +19,16 @@
     "xorshift": "^0.2.0"
```

Comment:
- lodash can be removed
- node-int64 will be removed in another diff
- assert is not needed
- request is for making http calls in the sampler